.github/ISSUE_TEMPLATE/model-compression-bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Create a model compression/speedup issue
+about: Report an issue or question while using nni instance (deployment).
+
+
+---
+
+**Describe the bug**:
+
+
+
+**Environment**:
+- NNI version:
+- Training service (local|remote|pai|aml|etc):
+- Python version:
+- PyTorch version:
+- Cuda version:
```

Comment:
use`cpu or cuda version`?

Reply:
So need to add a new line `- Cup version:` or change `- Cuda version` to `- Cpu or cuda version`

Reply:
i think it's better to change it to ` Cpu or cuda version`